.circleci/config.yml

```diff
@@ -631,7 +631,7 @@ workflows:
             - docs-linkcheck
           filters:
             branches:
-              only: master
+              only: staging
```

Comment:
Can you elaborate on this?  I don't even see that we have a `staging` branch...

Reply:
That's the branch bors uses for merging
(It takes "current master" and a PR, merges that together in "staging" and lets CI run, if that is green it gets merged into master).

In theory we don't need to let tests run on master at all anymore, though disabling that makes the config really bloated (I need to add a filter to _every single job_).
Running the deploy on staging has the advantage that if it fails, bors will complain and not merge, thus allowing a user to catch it and retry (failures on master have the tendency to get lost, because the only way to see it is by the small checkmark on the top of the page or the badge in the readme)

Reply:
Thanks for explaining.  That's cool.

(Someday we might only want to deploy docs from tagged commits, but that's an entirely separate discussion).

Reply:
That one can be done.
Maybe even have docs from both master and tagged commits, e.g. https://mozilla.github.io/glean/master/book/index.html and https://mozilla.github.io/glean/v21.1.1/book/index.html

and some redirect stuff (though that gets a bit hard as we deploy on github pages)